install_config/syncing_groups_with_ldap.adoc

```diff
@@ -865,30 +866,32 @@ users: <5>
 <5> The users that are members of the Group, named as specified by the sync file.
 ====
 
-[[nested-sync]]
+[[sync-ldap-nested-example]]
 == Nested Membership Sync Example
 
-Groups in OpenShift do not nest.  Thus the LDAP server must flatten group membership before
-the data can be consumed.  Microsoft's Active Directory Server supports this feature via the
-https://msdn.microsoft.com/en-us/library/aa746475(v=vs.85).aspx[LDAP_MATCHING_RULE_IN_CHAIN]
-which has the OID `1.2.840.113556.1.4.1941`.  Furthermore, only explicitly
-xref:../install_config/syncing_groups_with_ldap.adoc#running-ldap-sync[whitelisted] groups
-can be synced when using this matching rule.
+Groups in {product-title} do not nest. The LDAP server must flatten group
+membership before the data can be consumed. Microsoft's Active Directory Server
+supports this feature via the
+link:https://msdn.microsoft.com/en-us/library/aa746475(v=vs.85).aspx[`LDAP_MATCHING_RULE_IN_CHAIN`]
+rule, which has the OID `1.2.840.113556.1.4.1941`. Furthermore, only explicitly
+xref:../install_config/syncing_groups_with_ldap.adoc#running-ldap-sync[whitelisted]
+groups can be synced when using this matching rule.
 
-This section has an example for the augmented Active Directory schema which synchronizes a
-group named *admins* that has one user *Jane* and one group *otheradmins* as members.
-The *otheradmins* group has one user member: *Jim*.  This example explains:
+This section has an example for the augmented Active Directory schema, which
+synchronizes a group named `admins` that has one user `Jane` and one group
```

Comment:
@enj We had a style guideline change about wrt backtick usage since this topic first began life ([current markup guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference)). I can go through the rest of it and get things better in sync.